docs/src/main/sphinx/sql/set-path.rst

```diff
@@ -0,0 +1,51 @@
+========
+SET PATH
+========
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    SET PATH path-element, path-element, ...
+
+Description
+-----------
+
+Update the session to define a collections of paths to a functions or table
+functions in a catalog. Each path-element uses a period-separated syntax of to
+specify the catalog name and schema location ``<catalog>.<schema>``of the
+function or only the schema location ``<schema>`` in the current catalog. The
+current catalog is set with :doc:`use` or as part of a client tool connection.
+Catalog and schema must exist. Function names must be unique across all path
```

Comment:
This is a guess @martint  .. it could also be that whatever is first in the path wins .. Specific examples I am thinking of is two path elements that point to two catalogs that use the same connector .. and both have the query table function in the same system schema for example.

Give that the query table function also exists in multiple connectors it could even different connectors and catalogs.